test/controllers/contactPreference/EmailToUseControllerSpec.scala

```diff
@@ -220,6 +226,65 @@ class EmailToUseControllerSpec extends ControllerBaseSpec {
             messages(Jsoup.parse(bodyOf(result)).title) shouldBe internalServerErrorTitle
           }
         }
+
+        "User has an unverified email address" when {
+
+          "a successful verification attempt is made" should {
+
+            lazy val result = {
+              mockConfig.features.letterToConfirmedEmailEnabled(true)
+              mockIndividualAuthorised()
+              mockUnverifiedEmail()
+              mockCreateEmailVerificationRequest(Future.successful(Some(true)))
+              target().submit(yesRequest)
+            }
+
+            "return 303" in {
+              status(result) shouldBe Status.SEE_OTHER
+            }
+
+            "redirect the user to the manage-vat overview page" in {
```

Comment:
This line & 266 both need updated with correct page name (it's just the names, the tests themselves seem fine)